db/schema.rb

```diff
@@ -71,8 +71,8 @@
   end
 
   create_table "subscriber_lists", id: :serial, force: :cascade do |t|
-    t.string "title", limit: 255
-    t.string "gov_delivery_id", limit: 255
+    t.string "title"
```

Comment:
Do you know why these changed?

Reply:
Yep, I did some digging around. I've added a commit to explicitly set these:
https://github.com/alphagov/email-alert-api/pull/230/commits/bca15f51e92167ad687f3c2eb8d0e7ccd23ca0f5